core/model/src/main/java/org/eclipse/rdf4j/model/datatypes/XMLDatatypeUtil.java

```diff
@@ -676,6 +681,15 @@ public static boolean isValidAnyURI(String value) {
 		}
 	}
 
+	public static boolean isValidLanguage(String value) {
```

Comment:
I seem to remember we had a different util method for this somewhere else. Can't remember the details though @ansell do you know what I'm thinking of?